templates/add_blurb.html

```diff
@@ -18,10 +18,10 @@ <h3>📜🤖 Blurb it again?</h3>
                 <input type="hidden" name="csrf" value="{{ csrf }}">
                 <div class="mb-3">
                     <div class="col form-inline">
-                        <label for="bpo_number" class="form-inline">bpo-</label><input type="text" class="form-control form-inline" placeholder="12345" id="bpo_number" name="bpo_number" required>
+                        <label for="issue_number" class="form-inline">gh-</label><input type="text" class="form-control form-inline" placeholder="12345" id="issue_number" name="issue_number" required>
```

Comment:
Is there a reason why this `gh-` is lowercase and the next one (for the PR) is uppercase?.

Reply:
The `gh-` becomes part of the filename which is lowercased:
Example: `Misc/NEWS.d/next/Security/2022-03-15-10-31-10.gh-123.YOkPty.rst`.
But no big deal, if it's better to be uppercased, I can change it.

Reply:
In the filename lowercase is fine -- I was referring to that specific line that shows up in the form (you can also see it in the screenshot you posted above).  No big deal either way, just curious if there was some reason I missed.

Reply:
There was no specific reason. I was simply replacing lowercase `bpo` with lowercase `gh`.
However I do think we can do better.

I was thinking to replace these with `GH Issue #` and `GH PR #`. Will this work for you?

Reply:
I've changed the form labels.
Since I've changed the blurb CLI to use `gh-issue-<n>` in the filename, I've adjusted this PR to do the same.